### DISCUSSION TITLE
Pagination to show only arrows

### DISCUSSION THREAD

**Discussion by darcimm** [twilio-labs/paste#2858] [Office hours] [answered]
We're adding pagination to an SQL jobs page, however the engineering team would like to reduce the component by removing the pages and only showing left/right arrows.

Original component
<img width="255" alt="image" src="https://user-images.githubusercontent.com/83974127/205687993-05cad28c-fa55-43a0-8ac5-6697b923e41a.png">

Proposed
![image](https://user-images.githubusercontent.com/83974127/205688323-be2a570d-c579-4bd2-8f4c-188d3118043e.png)

The reason they want to do this is "_because to display the numbers for pagination we have to get all the sql jobs from the database in the backend. If we only show the arrows we can just fetch 10 entries at a time. I think this would improve the performance loading time on the app for the user_."

Do you have any recommendations to improve performance of the original pagination component? If not, is displaying only arrows a good workaround? Any other ideas? Thanks!!

**Comment by brannytempy** [comment, marked as answer]
Hi @darcimm 👋

This breaks Evergreen's standards today, but I recommend you follow Paste's guidance. They support simple next/previous buttons.

For instances when you are progressively fetching items, you  [could also consider adding custom labels](https://paste.twilio.design/components/pagination#pagination-with-custom-labels)

![CleanShot 2022-12-05 at 11 30 33](https://user-images.githubusercontent.com/1590723/205690242-79590884-eca6-4566-8424-1c1fd35fcd87.png)

The example above shows how you could potentially have this be more clear to the user, while not showing the total amount.
